### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by alicera** [ultralytics/yolov5#6097]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

How to find the weight that I train is overfitting?

**Comment by iceisfun** [comment]
I tend to find it happens with too few samples and excessive training.

Usually I reach near 1.0 mAP quickly and end up with poor inference results, adding more samples and reducing training time almost always works well.

**Comment by alicera** [comment]
How to define excessive training?
If I already train 300 epoch with yolov5s.pt and name the weight to latst_300.pt.
Then I add new 5000 images to my dataset, and retrain it with latst_300.pt for 300 epoch.
Is the case excessive training ?

**Comment by alicera** [comment]
excessive training.
Can I though the loss or mAP to find  excessive training ?

**Comment by alicera** [comment]
If I already train a weight named yolov5_300.pt (300 epoch)
Then, I command --weights yolov5_300.pt, it will start from 0 epoch.
Let it as the initial weight and retrain the same data.
Is it the overfitting case if the mAP is better than yolov5_300.pt ?

**Comment by glenn-jocher** [comment]
@alicera overfitting just means your validation losses are no longer dropping and are increasing instead.

**Comment by alicera** [comment]
Because I do this, the mAP decrease. After the 50 epoch,it will increase.
I think it was affect by learning rate.

Do you have learning rate formula for yolov5?

**Comment by alicera** [comment]
> @glenn-jocher overfitting just means your validation losses are no longer dropping and are increasing instead.

1. Is the case overfitting that you said?
2. About 30 epoch, it was overfitting. (The val loss increase)
Or about 70 epoch, it was overfitting. (The val mAP decrease) 
Which is the correct?
![image](https://user-images.githubusercontent.com/59388989/151477086-4ed33d12-68e8-424d-ab07-4e720d527674.png)

**Comment by glenn-jocher** [comment]
Yes
